src/components/MapForm.js

```diff
@@ -211,15 +210,70 @@ export class MapForm extends Component {
         lng
       }
     }));
-    this.updateFields(lat, lng)
+    this.updateFields(lat, lng);
   }
 
   getAPIGraphData = (event) => {
     event.preventDefault()
-    fetch(`https://api.census.gov/data/timeseries/eits/${this.state.tableId}?get=cell_value,data_type_code,time_slot_id,error_data,category_code,seasonally_adj,geo_level_code&time=from+${this.state.fromYear}`)
+    fetch(`https://api.census.gov/data/timeseries/eits/${this.state.tableId}?get=cell_value,data_type_code,time_slot_id,error_data,category_code,seasonally_adj&time=from+${this.state.fromYear}`)
```

Comment:
Looks like `geo_level_code` was dropped here, can we add that back in?

Reply:
oops, added it back, thanks